test/AxelarDepositService.js

```diff
@@ -206,16 +251,33 @@ describe('AxelarDepositService', () => {
 
     it('should unwrap native currency', async () => {
       const recipient = userWallet.address;
-      const nonce = formatBytes32String(1);
+      const salt = formatBytes32String(1);
       const amount = 1e6;
 
+      const expectedDepositAddress = getCreate2Address(
+        depositService.address,
+        keccak256(
+          defaultAbiCoder.encode(
+            ['bytes32', 'bytes32', 'address'],
+            [
+              keccak256(toUtf8Bytes('deposit-withdraw-native')),
+              salt,
+              recipient,
+            ],
+          ),
+        ),
+        keccak256(`0x${DepositReceiver.bytecode}`),
+      );
+
       const depositAddress =
-        await depositService.depositAddressForWithdrawNative(nonce, recipient);
+        await depositService.depositAddressForWithdrawNative(salt, recipient);
+
+      expect(depositAddress).to.be.equal(expectedDepositAddress);
 
       await token.connect(ownerWallet).transfer(depositAddress, amount);
 
       await expect(
-        await depositService.withdrawNative(nonce, recipient),
+        await depositService.withdrawNative(salt, recipient),
       ).to.changeEtherBalance(userWallet, amount);
     });
```

Comment:
Maybe add some tests for name/symbol etc, if we don't have those?

Reply:
We are removing `IERC20Metadata` from our IERC20 interface. No need to test those